app/decorators/news_tematica/news_tematica_decorator.rb

```diff
@@ -20,43 +16,55 @@ def html_con_contadores
     def titulares
-      if source.tematica.try(:seccion_titulares).present?
-        q = contenido_class.where("#{ source.tematica.seccion_titulares } = true").where(created_at: source.fecha_desde..source.fecha_hasta).where(fecha_titulares: source.fecha_desde..source.fecha_hasta)
-      elsif source.tematica.try(:tag_id)
-        taggings = tagging_class.where(tag_id: source.tematica.tag_id).where(created_at: source.fecha_desde..source.fecha_hasta).where(fecha_titulares: source.fecha_desde..source.fecha_hasta)
+      if suscribible.try(:seccion_titulares).present?
+        q = contenido_class
+          .where("#{ suscribible.seccion_titulares } = true")
+          .where(created_at: fecha_desde..fecha_hasta)
+          .where(fecha_titulares: fecha_desde..fecha_hasta)
+      elsif suscribible.try(:tag_id)
+        taggings = tagging_class
+          .where(tag_id: suscribible.tag_id)
+          .where(created_at: fecha_desde..fecha_hasta)
+          .where(fecha_titulares: fecha_desde..fecha_hasta)
         q = contenido_class.where(id: taggings.map(&:taggable_id))
       else
-        q = contenido_class.where(created_at: source.fecha_desde..source.fecha_hasta).where(fecha_titulares: source.fecha_desde..source.fecha_hasta)
+        q = contenido_class.where(created_at: fecha_desde..fecha_hasta).where(fecha_titulares: fecha_desde..fecha_hasta)
       end
       prioriza q.publicado.in_locale('es').includes(:fotos, :veces_leido, :blog)
     end
 
     # Lo más leído se apoyará en los scopes de las portadas temáticas
     def lo_mas_leido
-      msgs = contenido_class.publicado.in_locale('es').includes(:veces_leido, :blog).where(created_at: source.fecha_desde..source.fecha_hasta)
-      msgs = msgs.send("de_#{ source.tematica.scope_lo_mas_leido }".to_sym) if source.tematica.try(:scope_lo_mas_leido).present?
+      msgs = contenido_class.publicado.in_locale('es').includes(:veces_leido, :blog)
+      msgs = msgs.where(created_at: fecha_desde..fecha_hasta)
+      msgs = msgs.send("de_#{ suscribible.scope_lo_mas_leido }".to_sym) if suscribible.try(:scope_lo_mas_leido).present?
       msgs.sort_by { |msg| 100 - msg.contador_veces_leido * msg.factor_corrector_para_nuevos }
     end
 
     # Los foros pueden ser foros principales (basados en subtipo_id) o foros temáticos (basados en tag_id)
     def temas
-      if source.tematica.try(:subtipo_id)
-        q = contenido_class.where(tema: source.tematica.subtipo_id).where(created_at: source.fecha_desde..source.fecha_hasta)
-      elsif source.tematica.try(:tag_id)
-        taggings = tagging_class.where(tag_id: source.tematica.tag_id).where(tema: Subtipo::ARRAY_FOROS_NORMALES).where(created_at: source.fecha_desde..source.fecha_hasta)
+      if suscribible.try(:subtipo_id)
+        q = contenido_class
+          .where(tema: suscribible.subtipo_id)
+          .where(created_at: fecha_desde..fecha_hasta)
+      elsif suscribible.try(:tag_id)
```

Comment:
`try?` is a bit more than `respond_to?`, because it could respond with false or nil, which is evaluated as true for `respond_to?`, but evaluated as true for `try?`.